assets/css/custom.css

```diff
@@ -424,7 +424,7 @@ body:last-child .btn-u, x:-moz-any-link {
 .browse-courses {
```

Comment:
Could we reuse this style in other places as well? If this is the case, a more generic name would be more appropriate than content related.